tests/PhpGeneratorBundle/RestClient/GenerateRestClientCommandTest.php

```diff
@@ -51,15 +51,32 @@ protected function setUp()
      * @dataProvider dataProviderTestGenerateCode
      *
      * @param string $apiName
+     * @param string|null $libraryName
+     * @param string|null $libraryVersion
+     * @param string|null $platformVersion
```

Comment:
these are redundant as already contained in method signature, IMO only dataProvider should remain